data_model.md

```diff
@@ -0,0 +1,449 @@
+---
+layout: datamodel
+permalink: /datamodel/
+---
+
+Data Model -- Draft
+===================
+{: .anchor}
+The goal of this document is to explain the Solar Forecast Arbiter data
+model. To keep the framework architecture simple and secure, the API
+closely follows this data model. Therefore, it is worthwhile for API
+users to understand the data model. Please see
+[*dev-api.solarforecastarbiter.org*](https://dev-api.solarforecastarbiter.org/)
+for the technical API documentation. The Dashboard should be clear
+enough that users of it do not need a complete understanding of the data
+model. The Dashboard may contain additional features and abstractions to
+expedite common use cases.
+
+Data ownership, access, and control issues will be discussed in other
+documents.
+
+To introduce the data model, consider a framework user who will upload
+observed data for a new site. The framework user performs the following
+steps in order:
+
+1.  Define a Site by name, location, and timezone, and additional
+    parameters for a Site representing a power plant (e.g. AC capacity,
+    DC capacity).
+
+2.  For each Observation,
+
+    a.  Define the Observation: name, variable (e.g., GHI), value type
```

Comment:
The inner list doesn't seem to actually be a list. Is that easy to fix? Fine to leave alone if not.

Reply:
I didn't catch this, but vanilla markdown doesn't support lettered lists.

Reply:
don't worry about it then.